.github/workflows/triage-labelled.yml

```diff
@@ -5,5 +5,7 @@ on:
         types: [labeled]
 
 jobs:
-  call-triage-labelled:
-    uses: vector-im/element-web/.github/workflows/triage-labelled.yml@develop
+    call-triage-labelled:
+        uses: vector-im/element-web/.github/workflows/triage-labelled.yml@develop
+        secrets:
+            ELEMENT_BOT_TOKEN: ${{ secrets.ELEMENT_BOT_TOKEN }}
```

Comment:
maybe just using inherit here would be more flexible in the long term https://docs.github.com/en/actions/using-workflows/reusing-workflows#:~:text=github/labeler.yml-,secrets%3A%20inherit,-Example%20reusable%20workflow

Reply:
Good call. Have made it so.